Tiltfile

```diff
@@ -83,11 +83,18 @@ local_resource(
     "evm-create-configs", "python3 evm_integration.py %s %s" % (rpc_url_anvil, ws_url_anvil), resource_deps=["evm-deploy-contracts"]
 )
 
+# svm resources
```

Comment:
this comment was meant to be for a section of the file which contains svm resources. But now since all of them have the svm prefix, we can just remove the comment